ovs/lib/mdsservice.py

```diff
@@ -636,7 +668,7 @@ def get_mds_storagedriver_config_set(vpool, check_online=False):
                     client.run('pwd')
                 except UnableToConnectException:
                     continue
-            mds_service, load = MDSServiceController.get_preferred_mds(storagerouter, vpool, include_load=True)
+            mds_service, load = MDSServiceController.get_preferred_mds(storagerouter, vpool)
```

Comment:
A TypeError error might be raised here when None is returned. It might be better to either validate the return value, or always return a tuple.